lib/reek/source/source_code.rb

```diff
@@ -38,9 +41,9 @@ def initialize(code, description, parser = Parser::Ruby22)
       def self.from(source)
         case source
         when File     then new(source.read, source.path)
-        when IO       then new(source.readlines.join, 'STDIN')
+        when IO       then new(source.readlines.join, IO_IDENTIFIER)
         when Pathname then new(source.read, source.to_s)
-        when String   then new(source, 'string')
+        when String   then new(source, STRING_IDENTIFIER)
         end
```

Comment:
Is this really better? We don’t use `IO_IDENTIFIER` nor `STRING_IDENTIFIER` anywhere else, so all this change means in practice is ‘I’ll have to look up what these two are every time I read the `SourceCode.from` method’ and ‘every time I encounter `'STDIN'` and I’ll grep for it I’ll find `SourceCode::IO_IDENTIFIER` and will have to grep for that next’.